test-script2

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
```

Comment:
Is this script intended to be here? If so it probably would be nice with a more descriptive name.

Reply:
Not suposed to be here, will remove.,